pydantic/main.py

```diff
@@ -342,72 +338,6 @@ def model_construct(cls: type[Model], _fields_set: set[str] | None = None, **val
             m.__pydantic_post_init__(context=None)
         return m
 
-    def _copy_and_set_values(self: Model, values: dict[str, Any], fields_set: set[str], *, deep: bool) -> Model:
-        if deep:
-            # chances of having empty dict here are quite low for using smart_deepcopy
-            values = deepcopy(values)
-
-        cls = self.__class__
-        m = cls.__new__(cls)
-        _object_setattr(m, '__dict__', values)
-        _object_setattr(m, '__fields_set__', fields_set)
-        for name in self.__private_attributes__:
-            value = getattr(self, name, Undefined)
-            if value is not Undefined:
-                if deep:
-                    value = deepcopy(value)
-                _object_setattr(m, name, value)
-
-        return m
-
-    # @typing_extensions.deprecated('This method is now deprecated; use `model_copy` instead')
-    def copy(
```

Comment:
I moved this and all other from-v1-deprecated methods to the bottom of the class in the same order they existed in v1.

I would also like to reorder the other (non-deprecated) methods of BaseModel, but will do that as a separate PR at an opportune time to reduce chances for merge conflicts etc.